.bashrc

```diff
@@ -24,3 +19,17 @@ alias l='ls $LS_OPTIONS -lA'
 # alias rm='rm -i'
 # alias cp='cp -i'
 # alias mv='mv -i'
+
+[ -z "$PS1" ] && return
+
+export PS1='\h:\w\$ '
+umask 022
+
+if shopt -q login_shell
```

Comment:
Yeah, it was part of trying to figure out why nothing was working in the "usual" ways.
In principle it allows one to set up the environment differently depending on what type/level of login is desired.
Ended up that I just made the `ssh` command ask for a login interactive shell, because nothing else that should have worked, was working.
Might be worth it to see if Jes can set things up to be more reliable in regards to `.bashrc`, which is currently *not* sourced when executing a command over `ssh` in this `petalinux` variant [see here](https://stackoverflow.com/questions/940533/how-do-i-set-path-such-that-ssh-userhost-command-works)